core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -48,6 +48,26 @@
    Map<Address, Response> invokeRemotely(Map<Address, ReplicableCommand> rpcs, RpcOptions options);
 
    /**
+    * Asynchronously sends the {@link ReplicableCommand} to the destination using the specified {@link DeliverOrder}.
+    *
+    * @param destination  the destination's {@link Address}.
+    * @param command      the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    */
+   void sendTo(Address destination, ReplicableCommand command, DeliverOrder deliverOrder);
```

Comment:
About the stats, I haven't think about it. Do you think it makes sense? IMO, not really. It should be similar than the average write time.
About the replication timeout, it is used to wait for backups acks. I think it is more inline with the previous algorithm...